PortfolioApp/UsersScreen/Users/UsersTableViewController.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+final class UsersTableViewController: UITableViewController {
+    public typealias ReusableType = UsersTableViewController
+    @IBOutlet private weak var usersTableView: UITableView!
+    var presenter = UsersPresenter()
```

Comment:
it should be only reference to presenter. You are already created the presenter in the Composer